inventory/group_vars/in.yml

```diff
@@ -0,0 +1,18 @@
+---
+
+# See https://github.com/openfoodfoundation/ofn-install/wiki/Setup for more info
+checkout_zone:
+country_code: IN
+currency: INR
+locale: en
+language: en
+language_packages:
+  - language-pack-en-base
+l10n_repo: https://github.com/duleorlovic/l10n_in.git
```

Comment:
I notice this is your fork. Technically it's ok to use your own fork, but I can see it's simply to fix a small error in the data, so I think it would be better to submit your update to the `openfoodnetwork` repo. 
I've merged your update and granted you access to the repo, so would you like to point to that repo?

Reply:
Great, I have updated the link so now it is OFN repo